wurst/event/EventHelper.wurst

```diff
@@ -69,3 +73,321 @@ public function eventid.isKeyboardEvent() returns boolean
 public function eventid.isMouseEvent() returns boolean
 	let eventId = this.getHandleId()
 	return (eventId >= 305 and eventId <= 307)
+
+/** This class exposes all Getters and Setters for trigger events.
+	Note: As these functions simply wrap natives, they will only return proper values during event callbacks (actions/conditions).
+	To use these values in a subsequent closure, save the event data in a variable at the time the event was fired. */
+public class EventData
```

Comment:
@Krypt0n @Donach I just noticed this now..
Is there any reason you went totally wild with the function names?
Often the naming is completely inverse of the native (just confusing ppl)
or it's even lacking original information, e.g. `getDialogButton` vs `GetClickedButton`.
I'm thinking about changing them on master as long as most people haven't adapted yet..

Reply:
"Is this stricly for autocomplete? or what"  Yes
- The idea was to have some consistency for trigger getters and namings, and also the hotdoc should include from which triggers events the getters can be used. Also one use case for this commit was (I think its now 6 months ago once we first talked about adding these things) to keep the inversed pattern logic from jass to wurst. Thus having more buttom-up and less top-down syntax. Its a bit of a point if you prefer predictive parsing or resursive descent parsing in the end.

I had no time yet to check the commit there. I will catch it up soon, and then come back with some proper namings and hotdoc.

And I agree, this does not seem to be of any use right now tbh ..